Meshing application not compiling in current master

As it says in the title. It is actually failing because of the CMake settings:
```
CMake Error at cmake_modules/pybind11Tools.cmake:131 (add_library):
  Cannot find source file:

    /home/jcotela/Kratos/applications/MeshingApplication/custom_python/add_custom_strategies_to_python.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx
Call Stack (most recent call first):
  applications/MeshingApplication/CMakeLists.txt:73 (pybind11_add_module)
```

There is no `add_custom_strategies_to_python.cpp` in the meshing application. Removing the line in CMakeFiles.txt that adds it to the application sources fixes the problem, but I prefer to leave the solution up to its developers (Hi @loumalouomega! :wave:)

--- a/applications/MeshingApplication/CMakeLists.txt
+++ b/applications/MeshingApplication/CMakeLists.txt
@@ -32,10 +32,10 @@ set( KRATOS_MESHING_APPLICATION_SOURCES
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_meshers_to_python.cpp 
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_processes_to_python.cpp 
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_utilities_to_python.cpp 
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_strategies_to_python.cpp 
+	 
 	${CMAKE_CURRENT_SOURCE_DIR}/external_includes/mesh_suites.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/local_refine_geometry_mesh.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/uniform_refine_utility.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/local_refine_geometry_mesh.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/uniform_refine_utility.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/metrics_levelset_process.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/internal_variables_interpolation_process.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/metric_fast_init_process.cpp
@@ -51,10 +51,10 @@ set( KRATOS_MESHING_APPLICATION_SOURCES
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_meshers_to_python.cpp 
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_processes_to_python.cpp 
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_utilities_to_python.cpp 
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_strategies_to_python.cpp 
+
 	${CMAKE_CURRENT_SOURCE_DIR}/external_includes/mesh_suites.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/local_refine_geometry_mesh.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/uniform_refine_utility.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/uniform_refine_utility.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/metrics_levelset_process.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/internal_variables_interpolation_process.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/metric_fast_init_process.cpp

